Content.Server/Theta/ShipEvent/Systems/ShipEventFactionSystem.cs

```diff
@@ -160,6 +172,19 @@ public override void Initialize()
         SubscribeLocalEvent<RoundRestartCleanupEvent>(OnRoundRestart);
     }
 
+    private void OnTeammateSpeak(EntityUid uid, ShipEventFactionMarkerComponent component, EntitySpokeEvent args)
+    {
+        if (args.Channel == null) return;
+        if (!EntityManager.HasComponent<WearingHeadsetComponent>(uid) && args.Channel.ID != "Common")
+            return;
+        if (component.Team == null) return;
```

Comment:
```suggestion
        if (args.Channel == null)
        	return;
        if (!EntityManager.HasComponent<WearingHeadsetComponent>(uid) && args.Channel.ID != "Common")
            return;
        if (component.Team == null)
        	return;
```
так красивее в одном стиле

Reply:
да и сам кондишн можно объединить и щаписать в несколько строк